ckanext/ontario_theme/schemas/internal/ontario_theme_dataset.json

```diff
@@ -129,6 +129,48 @@
             "en": "Municipality",
             "fr": "Municipalité"
           }
+        },
+        {
+          "value": "province",
+          "label": {
+            "en": "Province",
+            "fr": "Province"
+          }
+        },
+        {
+          "value": "wmu",
+          "label": {
+            "en": "Wildlife Management Unit",
+            "fr": "Unités de gestion de la faune"
+          }
+        },
+        {
+          "value": "fmu",
+          "label": {
+            "en": "Forest Management Unit",
+            "fr": "Unités de gestion forestière"
+          }
+        },
+        {
+          "value": "county",
+          "label": {
+            "en": "County",
+            "fr": "Comté"
+          }
+        },
+        {
+          "value": "district",
+          "label": {
+            "en": "District",
+            "fr": "District"
+          }
+        },
+        {
+          "value": "fsa",
```

Comment:
Keep in mind, it will be this value, not the label, that is displayed in the API.  e.g. `geographic_granularity: "fsa",`. I'd suggest in future going for clarity/self documenting values vs. short.